### PR TITLE
Add Optional telescope boresight offset & Fix tilt scaling issue between SW and LW segment offsets

### PR DESCRIPTION
**Pull request by Skyhawk172**
Fixed XTILT/YTILT header keyword description to micro radians.

Fixed 'get_segment_offset': docstring originally suggested returned values for
offsets were in pixels, but it turns out they are in arcseconds.  Because of
that confusion, the SW pixel scale was applied to LW data resulting in a scaling
of the offset.  This is now fixed and does not require pixel scale knowledge.

### CODE REVIEW

**Comment by Skyhawk172** [comment]
This is in response to https://github.com/spacetelescope/mirage/issues/457

File: mirage/psf/segment_psfs.py [hunk 1]
```diff
@@ -59,6 +59,10 @@ def generate_segment_psfs(ote, segment_tilts, out_dir, filters=['F212N', 'F480M'
     fov_pixels : int, optional
         Size of the PSF to generate, in pixels. Default is 1024.
 
+    boresight: list, optional
+        Telescope boresight offset in V2/V3. This offset is added on top of the individual
```
Comment (bhilbert4) [inline mirage/psf/segment_psfs.py]: Are users going to know that the input boresight offset values should be in units of arcmin? Or should that info be added here to the documentation?

**Comment by mperrin** [comment]
I agree, it seems like a good idea to add a label about the units for the offset. Especially because the code seems to use a mix of microradians, arcseconds, and arcminutes in different places! (Charles, do you think it might be worth switching the offset to be in arc seconds, so there’s one fewer different units mixed in? I’m OK with it either way, as long as it’s clear in the doc strings.)

**Comment by Skyhawk172** [comment]
I thought I had put the arcminutes in the docstring. I'll add it.

**Pull request merged**